includes/pages/guest_login.php

```diff
@@ -118,6 +119,10 @@ function guest_register()
             $email_news = true;
         }
 
+        if ($request->has('email_goody')) {
```

Comment:
Shouldn't the value be checked here? What if it is sent but the value is `false` or `0`?

Reply:
As it's a checkbox, browsers either send a value or don't send it at all so it's an ok shortcut imho